sqlserver/assets/configuration/spec.yaml

```diff
@@ -81,6 +81,18 @@ files:
       value:
         type: boolean
         example: false
+    - name: include_db_fragmentation_metrics
+      description: Include database fragmentation metrics.
+      value:
+        type: boolean
+        example: false
+    - name: object_name
```

Comment:
should it be a list?

Reply:
Agreed - it should be a list to handle multiple objects.

Also, could this be used for other metric types?  My guess is no, so we should also prefix it with `db_` to more clearly have it stand out from other options.